client/lib/plugins/notices.jsx

```diff
@@ -550,7 +550,83 @@ module.exports = {
 		}
 	},
 
+	additionalExplanation: function( error_code ) {
+		switch ( error_code ) {
+			case 'no_package':
+				return i18n.translate( 'Plugin doesn\'t exits in the plugin repo' );
+
+			case 'plugin_already_installed':
+				return i18n.translate( 'Plugin is already installed.' );
+
+			case 'incompatible_archive':
+				return i18n.translate( 'Incompatible Archive.' );
+
+			case 'empty_archive_pclzip':
+				return i18n.translate( 'Empty archive.' );
+
+			case 'disk_full_unzip_file':
+				return i18n.translate( 'Could not copy files. You may have run out of disk space.' );
+
+			case 'mkdir_failed_ziparchive':
+			case 'mkdir_failed_pclzip':
+				return i18n.translate( 'Could not create directory.' );
+
+			case 'copy_failed_pclzip':
+				return i18n.translate( 'Could not copy file.' );
+
+			case 'md5_mismatch':
+				return i18n.translate( 'The checksum of the files don\'t match.' );
+
+			case 'bad_request':
+				return i18n.translate( 'Invalid Data provided.' );
+
+			case 'fs_unavailable':
+				return i18n.translate( 'Could not access filesystem.' );
+
+			case 'fs_error':
+				return i18n.translate( 'Filesystem error.' );
+
+			case 'fs_no_root_dir':
+				return i18n.translate( 'Unable to locate WordPress Root directory.' );
+
+			case 'fs_no_content_dir':
+				return i18n.translate( 'Unable to locate WordPress Content directory (wp-content).' );
+
+			case 'fs_no_plugins_dir':
+				return i18n.translate( 'Unable to locate WordPress Plugin directory.' );
+
+			case 'fs_no_themes_dir':
+				return i18n.translate( 'Unable to locate WordPress Theme directory.' );
+
+			case 'fs_no_folder':
+				return i18n.translate( 'Unable to locate needed folder.' );
+
+			case 'download_failed':
+				return i18n.translate( 'Download failed.' );
```

Comment:
I wonder if this should be `no_package` instead? as we updated jetpack to use `no_package` for backwards compatibility.
https://github.com/Automattic/jetpack/blob/master/json-endpoints/jetpack/class.jetpack-json-api-plugins-install-endpoint.php#L42